build.gradle

```diff
@@ -454,3 +459,89 @@ configure(project.fineractJavaProjects) {
         }
     }
 }
+
+configure(project.fineractPublishProjects) {
+    apply plugin: 'maven'
+    apply plugin: 'maven-publish'
+    apply plugin: 'com.jfrog.bintray'
+
+    publishing {
+        publications {
+            mavenJava(MavenPublication) {
+                from components.java
+
+                versionMapping {
+                    usage('java-api') {
+                        fromResolutionOf('runtimeClasspath')
+                    }
+                    usage('java-runtime') {
+                        fromResolutionResult()
+                    }
+                }
+
+                pom {
+                    name = 'Fineract'
+                    description = 'A secure, multi-tenanted microfinance platform'
+                    url = 'https://fineract.apache.org'
+                    licenses {
+                        license {
+                            name = 'The Apache License, Version 2.0'
+                            url = 'http://www.apache.org/licenses/LICENSE-2.0.html'
+                        }
+                    }
+                    // TODO: should we only list the Fineract contributors here? Or everyone that sent a PR (ca. 150; a bit much)? Or should we just omit this?
```

Comment:
Let's simply omit this entire POM section entirely - remove?

Reply:
As far as I'm aware you have to provide a POM... otherwise it won't work on Maven Central... unless you meant something different?

We could reduce the information though and just have name, license, Git repo URL.

Reply:
As far as I'm aware you have to provide a POM... otherwise it won't work on Maven Central... unless you meant something different?

We could reduce the information though and just have name, license, Git repo URL.

Reply:
As far as I'm aware you have to provide a POM... otherwise it won't work on Maven Central... unless you meant something different?

We could reduce the information though and just have name, license, Git repo URL.

Reply:
Yeah sorry I was responding to your TODO, and meant specifically and only not listing developers at all, but keeping the rest of the POM. BTW how does this actually work - will the POM include the required `<dependencies>`? It, ideally, should, right?

Reply:
Yeah, the developer section didn't make too much sense to me neither. People can see this - if they are interested - here on Github a lot better.

As for the dependencies: yes, this is done automatically by the "maven" Gradle plugin. You get proper pom.xml with dependencies per module.

Reply:
... and it's gone